resources/views/admin/users/create.blade.php

```diff
@@ -0,0 +1,47 @@
+@extends('admin.main')
```

Comment:
Also if this file will be the same for update too, we will avoid code duplication